### PR TITLE
Added Mixpanel FAQ 

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Added Mixpanel FAQ describing API requests size limit

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

**Review by yashasvibajpai — APPROVED** [review]
Looks suitable!

File: docs/destinations/streaming-destinations/mixpanel.mdx [hunk 1]
```diff
@@ -382,6 +382,16 @@ This feature is available in the web device mode only.
 
 ### Can I pass `null` and empty values for the properties?
 
-Yes, RudderStack supports passing empty/`null` values for the properties sent to Mixpanel.
+Yes, RudderStack supports passing empty or `null` values for the properties sent to Mixpanel.
+
+### I am getting the 'Request Header Fields Too Large' error. What might be the reason?
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mixpanel.mdx:387]:
```suggestion
### Why am I getting the 'Request Header Fields Too Large' error?
```

File: docs/destinations/streaming-destinations/mixpanel.mdx [hunk 2]
```diff
@@ -382,6 +382,16 @@ This feature is available in the web device mode only.
...
+
+This error might occur due to the limits Mixpanel imposes on its API requests:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mixpanel.mdx:389]:
```suggestion
Mixpanel imposes certain limitations on its API requests such as:
```

File: docs/destinations/streaming-destinations/mixpanel.mdx [hunk 3]
```diff
@@ -382,6 +382,16 @@ This feature is available in the web device mode only.
...
+- The GET request URL above 19K characters (around 19 KB+) is blocked.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mixpanel.mdx:390]:
```suggestion
- Mixpanel blocks a `GET` request URL above 19K characters (around 19 KB+).
```

**Review by ameypv-rudder — APPROVED** [review]
Made some minor suggestions - looks good otherwise!

**Pull request merged**